Tasks/app-store-promote/app-store-promote.ts

```diff
@@ -52,25 +52,31 @@ async function run() {
         tl.debug('Read all inputs.');
 
         // Set up environment
-        let gemCache: string = process.env['GEM_CACHE'] || process.platform === 'win32' ? path.join(process.env['APPDATA'], 'gem-cache') : path.join(process.env['HOME'], '.gem-cache');
+        let gemCache: string = path.join(process.env['HOME'], '.gem-cache');
```

Comment:
we should check for process.env['GEM_CACHE'] and only use process.env['HOME']/.gem-cache if GEM_CACHE is not set